src/composables/ui.ts

```diff
@@ -31,6 +36,10 @@ const hiddenCards = useStorageRef<string[]>(
     ],
   },
 );
+const secureLogin = useStorageRef<ISecureLoginSettings>(
+  {},
+  STORAGE_KEYS.secureLogin,
+);
```

Comment:
In order to decrease the amount of future migrations and amount of separate storage entries consider moving the secure login settings to other settings.
I know that "other" is not the best name for these settings. Maybe in the future we could use something like `primarySettings` or `commonSettings`.